packages/flutter/lib/src/services/text_input.dart

```diff
@@ -650,6 +651,12 @@ abstract class TextInputClient {
 
   /// Updates the floating cursor position and state.
   void updateFloatingCursor(RawFloatingCursorPoint point);
+
+  /// Creates a [TextInputConfiguration] for this client.
+  ///
+  /// This configuration is requested when establishing or re-establishing the
+  /// connection between the [TextInput] and the engine.
+  TextInputConfiguration createConfiguration();
```

Comment:
This also looks like a breaking change? Implementors/Subclasses of this class will have to implement a new method?